connector-proxies/async-http/main.py

```
@@ -0,0 +1,132 @@
+        status = 0
+        url = params.get("url")
+        
+        basic_auth_username = params.get("basic_auth_username")
+        basic_auth_password = params.get("basic_auth_password")
+
+        if basic_auth_username and basic_auth_password:
+            auth = (basic_auth_username, basic_auth_password)
+
+        # TODO: error handling
+        http_response = await http_client.request(self.request_method, url,
+            headers=params.get("headers"),
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)